content/features/_index.md

```diff
@@ -6,9 +6,10 @@ date: 2021-10-15T11:02:11+01:00
 draft: false
 ---
 
-Rockstor is a Linux and BTRFS powered storage operating system with advanced NAS and Cloud Storage features.
+Rockstor is a Linux and BTRFS powered storage operating system with advanced NAS and Cloud Storage features, supporting
+multiple architectures like AMD64/ARM64EFI/Ten64 and more.
```

Comment:
```suggestion
multiple architectures: AMD64 & ARM64EFI.
```
Backing away from the Ten64 mention now given we have Traverse Technologies in our Partners section on the landing page: plus Ten64 is a platform, and an example of ARM64EFI. And we line up our future RISC-V compatibility: an inevitability as we update our base OS and dependencies. I know this looses our Ten64 mention and our Pi4 mention but we have Partner and Pic references prominent, respectively, for both.  Plus we have Pi4 (should add Pi5 once proven) & Ten64 specific downloads; and the Ten64 docs in turn have links back to us.